README.md

```diff
@@ -0,0 +1,26 @@
+# mapreduce
+A set of MapReduce programs to process brain images
+
+###Java implementation
+To run:<br/>
+<code>hadoop jar GetHistogram.jar GetHistogram <input file> <output folder></code>
+
+Dependencies: niftijio - https://github.com/cabeen/niftijio
+
+
+###Python implementation
+
+To run:<br/>
+<code>hadoop jar <hadoop streaming jar path> -input <input filename> -output <output directory> -mapper VoxelMapper.py -reduce VoxelCountReducer.py</code>
+
+
+Dependencies: nibabel - http://nipy.org/nibabel/
+
+---
+Note: Input file must be a text file containing the absolute file paths to all nii.tar.gz images.
+<br/> ex. filenames.txt
+<br/> /home/usr/img1.nii.tar.gz
+<br/> /home/usr/img2.nii.tar.gz
+
+For the python implementation, the input images found in the input text file currently need to be 2d slices of the 3d image.
```

Comment:
Now it can, but I just flattened the array. Not sure if it's better to keep the structure. Though I suppose it might not make much of a difference for a histogram count.